Documentation/integrations.md

```diff
@@ -6,8 +6,8 @@ This document tracks projects that integrate with flannel. [Join the community](
 
 [Kubernetes](https://kubernetes.io/docs/admin/networking/#flannel): Container orchestration platform with options for [flannel as an overlay](https://kubernetes.io/docs/admin/networking/#flannel).
 
-[Tectonic](https://coreos.com/tectonic/): CoreOS's enterprise ready, 100% pure-upstream distribution of Kubernetes.
+[Canal](https://projectcalico.docs.tigera.io/getting-started/kubernetes/flannel/flannel): Policy-based networking for cloud-native apps.
```

Comment:
This definition is weird. I would say: Kubernetes CNI plugin that uses Calico for network policies and intra-node communications and Flannel for inter-node communications.